.github/workflows/release.yaml

```diff
@@ -21,10 +21,12 @@ jobs:
             curl \
             jq \
             squashfs-tools \
-            xz-utils
+            xz-utils \
+            gawk
+
 
           images=(
-              "kubernetes-v1.28.2"
+              $(curl https://raw.githubusercontent.com/kubernetes/website/main/data/releases/schedule.yaml | yq -r '.schedules[].previousPatches[0].release' | awk '{print "kubernetes-v"$1}')
```

Comment:
Thanks, we need to move the curl call into it's own line for error handling, and also add some common curl options for robustness: `-fsSL --retry-delay 1 --retry 60 --retry-connrefused --retry-max-time 60 --connect-timeout 20`

I think we also need to filter out future `targetDate` entries, or?